### ISSUE TITLE
Problem when trying to update imported data

### ISSUE THREAD

**Issue by prog-toolbox** [strapdata/elassandra#179]
Elassandra : 5.5.0.10

JDK : 1.8

Hello,

After importing data from another cluster to my cluster through csv files (COPY TO & COPY FROM  mechanism), I tried to update these new records but nothing happens.
I am only able to retrieve data, but I cannot apply any modification on it.
The odd thing that I did this manipulation successfully yesterday, and I repeated the same steps earlier today and it does not work anymore.
Could you please explain this issue ?

I appreaciate your help.

**Comment by vroyer** [comment]
Hi
Could you give us a little bit of details about what is your schema, your mapping, your cvs file and what update does not work any more ?
Thanks in advance.

**Comment by prog-toolbox** [comment]
Hello,

My schema contains 43 parameters with different types (text, timestamp, map).
My mapping contains the corresponding types from Cassandra to Elasticsearch.
My csv was generated by the command : **COPY table_name TO /path_to_csv/table_name.csv;** and I imported it to my cluster by the command **COPY table_name FROM /path_to_csv/table_name.csv;**
The content of my csv file is confidential, so I apologize for not exposing the data.
As I said before, this test was performed successfully, yerterday, with the same conditions.
I activated the DEBUG mode on my API to see the generated queries by **org.springframework.data.cassandra**, and I noticed that all the CQL queries were generated correctly : 
''
 Executing CQL Statement [BEGIN BATCH UPDATE table_name SET param_name ........
''
I created a record following my schema and I updated it with no problems, so I am assuming that the issue is with the imported data.

Thank you.
